keywords.robot

```diff
@@ -396,8 +396,17 @@ Get DUT To Start State
     [Documentation]    Clear telnet buffer and get Device Under Test to start
     ...                state (RTE Relay On).
     Telnet.Read
-    ${result}=    Get Relay State
-    Run Keyword If    '${result}'=='low'    RteCtrl Relay
+    ${result}=    Get Power Supply State
+    Run Keyword If    '${result}'=='low'    Turn On Power Supply
+
+Turn On Power Supply
```

Comment:
The information about Sonoff should be taken from variables.robot or platform-configs. It should not be hardcoded that it is one of these two platforms.

Reply:
Changed